PyInstaller/hooks/hook-appdirs.py

```diff
@@ -0,0 +1,19 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013-2017, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+
+"""
+Import hook for appdirs.
+
+On Windows, appdirs tries 2 different methods to get well-known directories
+from the system: First with win32com, then with ctypes. Not using win32com here
+might avoid including Python's COM support altogether.
```

Comment:
I find the last sentence irritating - and it did irritate me. I suggest something like (taken from your pull-request message :-)
"Excluding win32com here avoids including all the win32com related DLLs in programs that don't include them otherwise."